### ISSUE TITLE
ResNet training from scratch instructions?

### ISSUE THREAD

**Issue by windweller** [pytorch/vision#270]
Hi,

I'm trying to use the model to train a ResNet-32 from ground up, but I'm a little bit less certain how to preprocess CIFAR-10 / CIFAR-100 data. Is there a recommended way to preprocess CIFAR to train a very good resnet model?

```python
dset.CIFAR10(pjoin(root, 'cifar10'), train=True, transform=None, target_transform=None, download=True)
```

For example, what should I pass into the `transform` here? Are there some information on some default hyperparameters and learning rate details I can find
